x/authenticator/ante/ante_test.go

```diff
@@ -140,13 +140,14 @@ func (s *AutherticatorAnteSuite) TestSignatureVerificationWithAuthenticatorInSto
 	}
 	feeCoins := sdk.Coins{sdk.NewInt64Coin(osmoToken, 2500)}
 
-	err := s.OsmosisApp.AuthenticatorKeeper.AddAuthenticator(
+	id, err := s.OsmosisApp.AuthenticatorKeeper.AddAuthenticator(
 		s.Ctx,
 		s.TestAccAddress[0],
 		"SignatureVerificationAuthenticator",
 		s.TestPrivKeys[0].PubKey().Bytes(),
 	)
 	s.Require().NoError(err)
+	s.Require().Equal(id, uint64(0), "Adding authenticator returning incorrect id")
```

Comment:
why are these 0? AFAICT authenticator ids should start at 1: https://github.com/osmosis-labs/osmosis/blob/f0d8ef1c1c6b5d66ae96c4bbdb8b45e6d55c8445/x/authenticator/keeper/keeper.go#L141-L156

Reply:
interesting... will have a look